src/work-allocation/containers/task-list/task-list.component.scss

```diff
@@ -60,31 +60,76 @@
 /**
  * The "Manage +" button should mirror the GDS accordion functionality. However,
  * it still needs to be in keeping with the styling within the table.
+ * EXUI-477 - Change from govuk-accordion to xui custom styling
  */
-.govuk-accordion__section-header {
+.xui-manage-header {
   padding-top: 5px;
 }
-.govuk-accordion__section-button {
-  &.xui-reset {
-    font-size: 19px;
-    margin-right: 20px;
-    text-decoration: underline;
-    font-weight: 400;
-    &:focus {
-      text-decoration: none;
-    }
-    .govuk-accordion__icon {
-      height: 14px;
-      width: 14px;
-      right: 0;
-      top: 15px;
-      margin-top: 0;
-    }
-    &[aria-expanded=true] {
-      .govuk-accordion__icon::after {
-        display: none;
-      }
-    }
+.xui-manage-button {
```

Comment:
I don't love the fact that you are declaring the same fairly complex style in 2 places. Can you move it to src/style/app.scss and import it?
https://dev.to/michaeljota/how-to-share-styles-between-angular-components-nhm#:~:text=Angular%20usually%20has%20a%20style,property%2C%20and%20using%20HostBinding%20decorator.